docs/_config.yaml

```diff
@@ -4,8 +4,8 @@ locale: "en-US"
 title: "The Kubernetes NMState project"
 description: A declarative API for Kubernetes node network management
 name: "The Kubernetes NMState team"
-url: "https://nmstate.github.io"
-baseurl: "/kubernetes-nmstate"
+url: "https://storage.googleapis.com"
```

Comment:
Why do you change this? Would not it break the page?

Reply:
This is from manual call to publish.sh, thanks !